x-pack/plugins/cases/public/components/create/owner_selector.tsx

```diff
@@ -35,7 +35,7 @@ interface Props {
   isLoading: boolean;
 }
 
-const DEFAULT_SELECTABLE_OWNERS = [SECURITY_SOLUTION_OWNER, OBSERVABILITY_OWNER] as const;
+const DEFAULT_SELECTABLE_OWNERS = Object.keys(OWNER_INFO) as Array<keyof typeof OWNER_INFO>;
```

Comment:
Maybe we could create a `OWNER_IDS` array in the constants file. Also, it would be nice to have explicit types for `OwnerId` and `OwnerInfo`, so we won't need to do this cast here.